docs/src/websocket_doc.md

```diff
@@ -0,0 +1,237 @@
+          "username": "string"    // Member user name
+        }
+      ],
+      "created_at": "string",     // Date and time of chat creation
+      "updated_at": "string"      // Date and time of the last chat update
+    }
+  ]
+}
+```
+
+---
+
+## createEmptyChat
+
+### Description:
+Create an empty chat entity, with no participants. The server creates a new chat and returns its identifier, as well as the timestamps of creation and last update.
+
+### Request:
+
+```json
+{
+  "type": "createEmptyChat",
+  "data": {
+    "name": "string"  // Chat Name (optional)
+  }
+}
+```
+
+### Response:
+
+```json
+{
+  "type": "emptyChatCreated",
+  "data": {
+    "id": "int",                 // New chat ID
+    "name": "string",            // Chat Name (if not specified, can be blank)
+    "participants": [],          // List of participants
+    "created_at": "string",      // Date and time of chat creation
+    "updated_at": "string"       // Date and time of the last chat update
+  }
+}
+```
+
+---
+
+## joinChat
+
+### Description:
+Add the current user to an existing chat room. Optionally specify the ID of the last read message so that the server knows when to count new messages for this user.
+
+### Request:
+
+```json
+{
+  "type": "joinChat",
+  "data": {
+    "chat": "int",                 // ID of the chat room the user is joining
+    "last_read_message": "int"     // ID of the last read message (optional)
+  }
+}
+```
+
+### Response:
+
+```json
+{
+  "type": "chatJoined",
+  "data": {
+    "chat": "int",                 // ID of the chat room the user has joined
+    "user": {
+      "id": "int",                 // user ID
+      "username": "string"         // User Name
+    },
+    "last_read_message": "int"     // ID of the last read message (if specified)
+  }
+}
+```
+
+---
+
+## getChatDetails
+
+### Description:
+Get detailed information about a specific chat, including all participants, creation date, and last update date.
+
+### Request:
+
+```json
+{
+  "type": "getChatDetails",
+  "data": {
+    "chatId": "int"  // chat ID
+  }
+}
+```
+
+### Response:
+
+```json
+{
+  "type": "chatDetails",
+  "data": {
+    "id": "int",                 // chat ID
+    "participants": [
+      {
+        "id": "int",             // participant ID
+        "username": "string"     // Member user name
+      },
+      {
+        "id": "int",             // participant ID
+        "username": "string"     // Member user name
+      }
+    ],
+    "created_at": "string",      // Date and time of chat creation
+    "updated_at": "string"       // Date and time of the last chat update
+  }
+}
+```
+
+---
+
+## newMessage
+
+### Description:
+Send a new message to the specified chat room.
+
+### Request:
+
+```json
+{
+  "type": "newMessage",
+  "data": {
+    "chat": "int",           // ID of the chat room to which the message is sent
+    "text": "string",        // Message Text
+    "is_read": "boolean"     // Message read status
```

Comment:
This field is optional as it defaults to False, please report this in the documentation